config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/ingredients", to: "ingredients#index"
+
+  get "/recipes/", to: "recipes#show"
```

Comment:
show pages usually have an ID associated, so the path would usually be `/recipes/:id`